packages/button/src/IconButtonV2.tsx

```diff
@@ -39,22 +39,23 @@ export const iconButtonStyle = ({ size, ...props }: IconButtonStyleProps) => css
   }
 `;
 
-export interface IconButtonProps extends ButtonProps {
+export interface IconButtonProps extends Omit<ButtonProps, 'shape'> {
   ['aria-label']: string;
+  /** Usually an icon from `'@ndla/icons'` */
+  children: ReactNode;
 }
 
 export const IconButton = ({
   type = 'button',
   size = 'small',
   colorTheme = 'primary',
   variant,
-  shape,
   fontWeight,
   inverted,
   children,
   ...rest
 }: IconButtonProps) => (
-  <button type={type} css={iconButtonStyle({ size, variant, shape, fontWeight, inverted, colorTheme })} {...rest}>
+  <button type={type} css={iconButtonStyle({ size, variant, fontWeight, inverted, colorTheme })} {...rest}>
```

Comment:
Hvorfor fjerne shape?

Reply:
Shape gjør ingenting per nå. Vi kan eventuelt gjøre det mulig å ha firkantede eller svakt avrundede iconButtons.

fontWeight kunne egentlig også vært fjernet, men dersom vi skal legge inn støtte for tekst ved siden av knappen kan det like gjerne bli værende

Reply:
Hm. Synes det kanskje er litt overkill å ha en prop for det?